lib/chip/README.md

```diff
@@ -0,0 +1,24 @@
+# CHIP
+Chip stands for Check-in Integration Point
+
+## Description
+Provide Veterans with a unified front door experience for preparing for and checking into their clinical appointments. Including, changing the process for outpatient clinical workflow that improves efficiency and decrease devices requiring publicly accessible shared surfaces that may increase the transmissions of communicable diseases
+
+## Design
+
+### CHIP settings and setup
+CHIP settings are stored at `config/settings.local.yml` under `chip_api_v2`. Initialize your setup using a client (or custom built) session key
+
+```
+check_in_session = CheckIn::V2::Session.build(data: { uuid: params[:id] }, jwt: low_auth_token)
```

Comment:
@lee-delarm6 this is for low auth check-in session. For this library, we instantiate `Service` object with the tenant params. Take a look at the Service class and how it is initialized.